src/patternfly/components/NotificationDrawer/notification-drawer.scss

```diff
@@ -62,10 +62,7 @@
   --#{$notification-drawer}__list-item-header-icon--MarginInlineEnd: var(--pf-t--global--spacer--sm);
 
   // List item header title
-  --#{$notification-drawer}__list-item-header-title--FontFamily: var(--pf-t--global--font--family--heading);
-  --#{$notification-drawer}__list-item-header-title--FontSize: var(--pf-t--global--font--size--heading--xs);
-  --#{$notification-drawer}__list-item-header-title--FontWeight: var(--pf-t--global--font--weight--heading--bold);
-  --#{$notification-drawer}__list-item-header-title--LineHeight: var(--pf-t--global--font--line-height--heading);
```

Comment:
Looks like we're still using this `--LineHeight` here, which we can remove https://github.com/patternfly/patternfly/blob/16cf11c1174c62157e2480faed8c1945a5bb65b2/src/patternfly/components/NotificationDrawer/notification-drawer.scss#L230